template.json

```diff
@@ -1,13 +1,12 @@
 {
   "package": {
     "dependencies": {
-      "bpk-component-button": "^6.0.1",
-      "bpk-component-code": "^3.1.3",
-      "bpk-component-text": "^6.1.4",
-      "bpk-mixins": "^28.0.0",
-      "bpk-stylesheets": "^7.1.26",
-      "react": "^16.4.2",
-      "react-dom": "^16.4.2"
+      "@skyscanner/backpack-react-scripts": "10.0.4",
```

Comment:
So we won't need this is here as this gets installed automatically with the BRS framework and setup itself :)

Reply:
Pushed an extra commit to remove this line 😎 thanks for explaining :)